custom.conf

```diff
@@ -1,3 +1,5 @@
+daemon off;
```

Comment:
what the goal of this?

Reply:
Docker will focus on 1 process as the main process, and nginx by default runs in a background process because of which it has issues with docker. Using daemon off, we can use it in foreground and prevent it to self daemonize. More info [here](https://stackoverflow.com/questions/18861300/how-to-run-nginx-within-a-docker-container-without-halting#:~:text=Use%20the%20daemon%20off%20configuration,.nginx.org%2FCoreModule.&text=To%20expand%20on%20Charles%20Duffy's,directly%20on%20the%20command%20line.)